comps/integrations/haystack/src/opea_haystack/__about__.py

```diff
@@ -0,0 +1 @@
+__version__ = "1.0.0"
```

Comment:
Just as a comment, having a pyproject.toml could suffice. For context, we usually don't have an `__about__.py` for our integrations: https://github.com/deepset-ai/haystack-core-integrations/blob/9dfa9bc096db2619e7bf7349e429b521862af1b1/integrations/amazon_bedrock/pyproject.toml#L38